parlai/crowdsourcing/tasks/fast_acute/run_no_self_chat.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Execute a Fast ACUTE run without model self-chats.
+"""
+
+import os
+import json
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Optional
+
+import hydra
+from mephisto.operations.hydra_config import register_script_config
+from mephisto.operations.registry import register_mephisto_abstraction
+from omegaconf import DictConfig, MISSING
+
+import parlai.crowdsourcing.tasks.fast_acute.run as acute_eval
+from parlai.crowdsourcing.tasks.acute_eval import run
+from parlai.crowdsourcing.tasks.fast_acute.run import (
+    FastAcuteBlueprint,
+    FastAcuteBlueprintArgs,
+    FastAcuteExecutor,
+)
+from parlai.crowdsourcing.tasks.fast_acute.util import FAST_ACUTE_TASK_DIRECTORY
+from parlai.crowdsourcing.utils.mturk import MTurkRunScriptConfig
+from parlai.utils.strings import normalize_reply
+
+BLUEPRINT_TYPE = "fast_acute_no_self_chat"
+
+
+@dataclass
+class FastAcuteNoSelfChatBlueprintArgs(FastAcuteBlueprintArgs):
+    _blueprint_type: str = BLUEPRINT_TYPE
+    _group: str = field(
+        default="FastAcuteNoSelfChatBlueprint",
+        metadata={
+            'help': "Execute a Fast ACUTE run without launching model self-chats"
+        },
+    )
+    config_path: str = field(
+        default=MISSING,
+        metadata={'help': 'Path to JSON of model types and their parameters'},
+    )
+
+
+@register_mephisto_abstraction()
+class FastAcuteNoSelfChatBlueprint(FastAcuteBlueprint):
+    """
+    Subclass of FastAcuteBlueprint without launching model self-chats first.
+    """
+
+    ArgsClass = FastAcuteNoSelfChatBlueprintArgs
+    BLUEPRINT_TYPE = BLUEPRINT_TYPE
+
+
+class NoSelfChatFastAcuteExecutor(FastAcuteExecutor):
```

Comment:
why is this additional script necessary? in the OSS fast acute if you specify the path to the model chats in the config file it'll automatically skip the self-chat in the standard run.py

Reply:
This is an artifact of the internal version of the Fast ACUTE code, and the two scripts can likely be merged by combining Mephisto blueprints. Will do this in an upcoming PR